content/docs/studio/rest-api.md

```diff
@@ -103,6 +103,144 @@ Running this example will fail because it requires that you have [set up remote
 
 </admon>
+</toggle>
+
+## Create new project
+
+To create a new project programmatically, you can use with the DVC Studio API.
+The API provides a POST request endpoint for creating projects. Authentication
+is required using a token, which can be generated from the user's profile
+settings in DVC Studio
+
+```yaml
+Endpoint: api/create-project
+HTTP Method: POST
+Content-Type: application/json
+```
+
+### Request
+
+Here are the possible values for each parameter in the request json:
+
+- `repo_url` (required) - the URL of the repository where the project will be
+  created. This field is mandatory.
+- `public` (boolean, default: `false`) - specifies whether the project should be
+  public (`true`) or not (`false`).
+- `team_name`(string) - the name of the team associated with the project if we
+  want to create the project in team space. This field is optional. If team name
+  is not specified, the project will be in user namespace.
+- `subdir`(string) - the subdirectory within the repository where the project
+  will be located, in case of a monorepo. This field is optional.
+- `name` (string) - the name of the project to be defined. If no name is
+  specified, it will be extracted from repository name. This field is optional.
+
+The request should contain following header for authorization.
+
+| header        | desc            | example value |
+| ------------- | --------------- | ------------- |
+| Authorization | Header for auth | token abc123  |
+
+Accepted formats for `repo_url` are:
+
+- `git@github.com:amritghimire/example-dvc-experiments.git`
+- `https://github.com/nimdraugsael/example-dvc-experiments.git`
+- `https://gitlab.example.org/amritghimire/example.git` (For custom gitlab)
+- `ssh://git@gitlab.com/xxx/yyy/zzz/repo.git`
+
+Similar format is supported for Github, Bitbucket and Gitlab.
+
+### Response
+
+Here is an example JSON response from the API request:
+
+```json
+{
+  "projects": [
+    {
+      "id": "tsanhawn9y",
+      "url": "https://studio.iterative.ai/user/amritghimire/projects/example-get-started-tsanhawn9y",
+      "name": "example-get-started"
+    }
+  ]
+}
+```
+
+This response confirms that the project was successfully created and provides
+the necessary information about the created project:
+
+- `id`: The unique identifier of the project. In this case, the project ID is
+  "tsanhawn9y".
+- `url`: The URL of the created project. You can access the project using this
+  URL. In this example, the project URL is
+  `https://studio.iterative.ai/user/amritghimire/projects/example-get-started-tsanhawn9y`.
+- `name`: The name of the created project. In this example, the project name is
+  "example-get-started".
+
+This information can be used to access and reference the project in the future.
+
+### Example
+
+First, you need your [DVC Studio client access token] . For this example, we set
+it in the `DVC_STUDIO_TOKEN` environment variable:
+
+```sh
+export DVC_STUDIO_TOKEN=<TOKEN>
```

Comment:
start with $

use cli instead of sh

Reply:
> use cli instead of sh

Not sure what the difference is. But updated.